.gitignore

```diff
@@ -1,2 +1,4 @@
 /build/
-/dist/
\ No newline at end of file
+/dist/
+/nbproject/private/
+/doc/
```

Comment:
Actually I misspell it in the `ant` script and the PR too... Must have copy paste it... I'll fix it right away